pkg/annotations/annotations.go

```diff
@@ -106,6 +109,33 @@ func ParseAnnotations(annotations map[string]string, clusterName string) (*Annot
 	return a, nil
 }
 
+func (a *Annotations) setAttributes(annotations map[string]string) error {
+	var attrs []*elbv2.LoadBalancerAttribute
+	var badAttrs []string
+	rawAttrs := util.NewAWSStringSlice(annotations[attributesKey])
+
+	for _, rawAttr := range rawAttrs {
+		parts := strings.Split(*rawAttr, "=")
+		switch {
+		case *rawAttr == "":
+			continue
+		case len(parts) < 2:
```

Comment:
You'll always expect `parts` to have length `2` here right?

Possibly best to set this check such that when it's anything other than 2, we return the error. Preventing cases like `attr=value=value2`